packages/bff/src/routers/catalogRouter.ts

```diff
@@ -102,15 +102,43 @@ const catalogRouter = (
     )
     .delete(
       "/eservices/:eServiceId/descriptors/:descriptorId/documents/:documentId",
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromBffAppContext(req.ctx, req.headers);
+        try {
+          await catalogService.deleteEServiceDocumentById(
+            unsafeBrandId(req.params.eServiceId),
+            unsafeBrandId(req.params.descriptorId),
+            unsafeBrandId(req.params.documentId),
+            ctx
+          );
+          return res.status(204).json().send();
+        } catch (error) {
+          const errorRes = makeApiProblem(error, emptyErrorMapper, ctx.logger);
```

Comment:
Since #745  we can add a specific log here as [we already do in scala](https://github.com/pagopa/interop-be-backend-for-frontend/blob/0d5b42922e9c49f5e0ef4d4e8ef368025e516637/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala#L776)